### ISSUE TITLE
How to destroy, detach, remove fliparoo instance?

### ISSUE THREAD

**Issue by rrolla** [Lullabot/fliparoo#44]
#### I want to destroy instance after every flip, when back to normal position.

Than initialize again, with new content.
##### In short: I want periodically flip images, but all content always are new and came from ajax.

Is this possible?

Thanks.
